bpf/Makefile

```diff
@@ -134,7 +135,8 @@ XDP_OPTIONS = $(LB_OPTIONS) \
 	-DDISABLE_LOOPBACK_LB:-DENABLE_NODEPORT_ACCELERATION:-DENABLE_IPV4:-DENABLE_IPV6:-DENABLE_NODEPORT:-DENABLE_DSR:-DDSR_ENCAP_MODE:-DDSR_ENCAP_GENEVE:-DDSR_ENCAP_IPIP=2 \
 	-DDISABLE_LOOPBACK_LB:-DENABLE_NODEPORT_ACCELERATION:-DENABLE_IPV4:-DENABLE_IPV6:-DENABLE_NODEPORT:-DENABLE_DSR:-DENABLE_CAPTURE:-DDSR_ENCAP_MODE:-DDSR_ENCAP_NONE:-DDSR_ENCAP_IPIP=2 \
 	-DDISABLE_LOOPBACK_LB:-DENABLE_NODEPORT_ACCELERATION:-DENABLE_IPV4:-DENABLE_IPV6:-DENABLE_NODEPORT:-DENABLE_DSR:-DENABLE_CAPTURE:-DDSR_ENCAP_MODE:-DDSR_ENCAP_IPIP:-DENABLE_SCTP:-DDSR_ENCAP_NONE=2 \
-	-DDISABLE_LOOPBACK_LB:-DENABLE_NODEPORT_ACCELERATION:-DENABLE_IPV4:-DENABLE_IPV6:-DENABLE_NODEPORT:-DENABLE_DSR:-DENABLE_CAPTURE:-DDSR_ENCAP_MODE:-DDSR_ENCAP_GENEVE:-DENABLE_SCTP:-DDSR_ENCAP_IPIP=2
+	-DDISABLE_LOOPBACK_LB:-DENABLE_NODEPORT_ACCELERATION:-DENABLE_IPV4:-DENABLE_IPV6:-DENABLE_NODEPORT:-DENABLE_DSR:-DENABLE_CAPTURE:-DDSR_ENCAP_MODE:-DDSR_ENCAP_GENEVE:-DENABLE_SCTP:-DDSR_ENCAP_IPIP=2 \
+	-DDISABLE_LOOPBACK_LB:-DENABLE_IPV4:-DENABLE_IPV6:-DENABLE_IPSEC:-DENABLE_IPSEC_RPS:-DENABLE_CPU_MAP:-DFROM_HOST: \
```

Comment:
Can do! Do you have guidance on how you'd like to see this done? We should be able to enable this on each kernel version but I'm not sure which config file(s) under each version to add the macro to.

Reply:
For a given program and a given kernel version, we typically have several files because some features are incompatible between them. Because of that, they are sort of local maximums for the complexity if you will (in the N+1 dimensional space where each of the N features is a dimension). So each file is trying to be one of the local maximums. More concretely, if ex. featA and featB are incompatible, we'd have one set of files maximizing complexity for featA and another for featB.

All that to say, in your case, AFAICS, the feature is not incompatible with anything but requires IPsec, so I'd add it to all files that have ENABLE_IPSEC, for the BPF programs that matter here.

Reply:
Oh ok that makes a ton of sense, thanks for the explanation! I'll add it to to places where we have IPsec, but the feature is also incompatible with tunnel mode routing (if I remember correctly, when we enable tunnel mode routing and IPsec, then try to load the XDP program it leads to an import of an skb-only library), so I'll avoid places where that's enabled too.